### PR TITLE
Updates to UI, documentation, and various methods

### PR DESCRIPTION
**Pull request by tannerblair**
I think this is going to work!

### CODE REVIEW

**Comment by rajsite** [comment]
Sorry for taking so long to start looking at this, finally got a free weekend not involving moving or vehicle problems

I started looking through the VIs but I'm realizing I don't remember what I should focus on. It looks like some changes we talked about were already done (ie sticking with 4 sided connectors, etc).

Excluding the feedback below, can you give me a high-level overview of changes. ie VIs in folder JSON Parsing just have icon / description changes, VIs X, Y, Z have error handling added, fixed some bug in VI Q. Doesn't have to be super specific just something to ground me looking through the changes. Also I need to get LVCompare setup with my git tools, if you have a favorite workflow let me know! :)

Some late early notes:
* Let's use LabVIEW 2014 SP1 as the source version and vipm 2017 (please mass compile to lv 2014 sp1)
* The vibp says it requires LabVIEW 2015 or later when I open it, save it to target LabVIEW 2014
* Exclude the `docs` folder from the vibp (needs a file server so not helpful to install locally)
* These are my recommendations for the Display Information:
Company Name: rajsite
Note: all lowercase
Copyright: Copyright (c) 2017, Milan Raj and Tanner Blair
Author name: rajsite
Note: all lowercase
License: MIT License
URL: Product Homepage: https://github.com/rajsite/i3-json

* Destination > Toolkit VIs change target to rajsite\i3-json
Note all lowercase and hyphen instead of space

* Update License.txt copyright  to say Copyright (c) 2017, Milan Raj and Tanner Blair

^ Let's discuss display info, destination, and license offline if needed :)

* Remove Test.vi as it's empty. After the initial commit will add some testing before actually publishing a new package
* In the README.md add a section for the Error Codes that describes the error code range also include a list of current used error codes, the LabVIEW message, and optionally some troubleshooting steps
ie
```
## Error Codes (53890 - ???)
- 53890 `Not Found Error`: Occurs when an item is not found in an object
- 53895 `Some other googleable part of the error string`: Clearer reason and solutions
```

* Change the `Error` folder to `Error Handling` on disk and in the project. Also change the layout to match the other polymorphic VI conventions.
ie.
/Error Handling/Error.vi
/Error Handling/Error Handling Files/Error - Create Error.vi
/Error Handling/Error Handling Files/Error - Invalid Type Error.vi
etc

**Comment by tannerblair** [comment]
@rajsite, these changes are updated and committed.

File: README.md [hunk 1]
```diff
@@ -3,3 +3,11 @@
 The i3 JSON Toolkit is used to parse and generate arbitrary JSON data in LabVIEW. The toolkit allows the user to manipulate JSON data following the specification described at json.org and in RFC 4627.
 
 An API interface consisting of a set of low-level and fast VIs are provided for the development of applications as well as a set of high-level and slow VIs to enable rapid prototyping. Utilities for visualizing JSON data in LabVIEW Trees are made available for user interaction.
+
+The i3 JSON Toolkit uses the following error codes:
+538900	Parse Error			This is thrown when the JSON String parse fails
```
Comment (rajsite) [inline README.md]: Use either Github Markdown Table layout or a Markdown list layout.

**Comment by rajsite** [comment]
@tannerblair please update the README as described above for a table / list layout of errors and delete Test.vi

I'll take a closer look tonight

**Comment by rajsite** [comment]
Pushed minor changes for the build output dir and standardized the product name to `i3-json` instead of mixing usage of `i3 JSON`

Ran my tests locally and everything looks good. There aren't "binary interface" changes but the new error codes are a breaking api change. So if everything looks good to you I'll pull the change, add some release notes and info to README, and make a major release.

**Comment by tannerblair** [comment]
Sounds good! Thanks for your patience and guidance on working out the issues.

**Pull request merged**